docs/pipeline.md

```diff
@@ -28,7 +28,7 @@ Automate process of uploading apps to IEM with Jenkins using shell script or wit
 *Prerequisites:*\
 *- IEM installed and configured*
 
-1) Go to the "Applications" section of your Edge Management. 
+1) Go to the "App-Projects" section of your Edge Management. 
 
 2) Click on "Create Project" button in the right upper corner. 
 
```

Comment:
Please update Videos on line 45 and 62. They seemed to be very old version of IEM.